howto/download_weights.md

```diff
@@ -10,6 +10,7 @@ git clone stabilityai/stablelm-base-alpha-3b checkpoints/stabilityai/stablelm-ba
 Or if you don't have `git-lfs` installed:
```

Comment:
Is the git clone command above correct?

Reply:
No it isn't, it should be the whole URL https://huggingface.co/stabilityai/stablelm-base-alpha-3b